payjoin-cli/src/app/v2.rs

```diff
@@ -300,6 +305,43 @@ impl App {
     }
 }
 
+async fn get_ohttp_keys(config: &AppConfig) -> Result<payjoin::OhttpKeys> {
```

Comment:
I feel that `get_ohttp_keys`, `fetch_ohttp_keys` and `normalize_proxy_url` should be part of the `payjoin` crate.
the `fetch..` could require the http client the user is using as input.

Reply:
I agree that there should be library code for fetching ohttp keys, however, the strong typing is already part of `payjoin::OhttpKeys::decode(bytes)`. That leaves the i/o. Since `payjoin` is a no-IO crate, this makes the functionality, including normalize_proxy_url as candidates for a payjoin-io crate. I'm happy to begin that work but would prefer to leave that to a new PR. It would be simple to create a dependency on a local payjoin-io crate as a follow up.

`get_ohttp_keys` is an app method that takes config as input, so I'm not sure that can be included in a common library.